### DISCUSSION TITLE
How to set KMESH_INTERPOLATE?

### DISCUSSION THREAD

**Discussion by gangliu-github** [ttadano/alamode#199] [Q&A] [answered]
I know the KMESH_INTERPOLATE should be commensurate with the supercell size of the harmonic force constant. If the supercell is 2 2 2, can the KMESH_INTERPOLATE be set to 4 4 4 or larger?

**Comment by ttadano** [comment, marked as answer]
When the harmonic force constants are computed using a 2x2x2 supercell, `KMESH_INTERPOLATE` should be 2x2x2 or 1x1x1 (or anisotropic settings such as 1x1x2 are also possible for anisotropic systems).

You can also set `KMESH_INTERPOLATE = 4 4 4`, and the code runs without raising an error, but I do not recommend it. If you want to set `KMESH_INTERPOLATE` to 4x4x4 or denser, I recommend that you compute the harmonic force constants using the corresponding size of the supercell.

**Reply by gangliu-github** [reply]
I see. Thanks a lot!
